src/lib/lss.ts

```diff
@@ -239,3 +246,144 @@ export async function createSplitsXml(config: Config): Promise<string> {
     }
   );
 }
+
+export function importSplitsXml(str: string): Config {
+  // xml parse
+  const parser = new DOMParser();
+  const xmlDoc = parser.parseFromString(str, "text/xml");
+  // GameName, CategoryName -> gameName, categoryName
+  const gameName =
+    xmlDoc.getElementsByTagName("GameName")[0].textContent?.trim() || "";
+  const categoryName =
+    xmlDoc.getElementsByTagName("CategoryName")[0].textContent?.trim() || "";
+  // Metadata Variables -> variables
+  const variablesVariables = xmlDoc
+    .getElementsByTagName("Variables")[0]
+    .getElementsByTagName("Variable");
+  let variables: Record<string, string> | undefined;
+  if (variablesVariables && variablesVariables.length > 0) {
+    // vs is mutated as an object, but not as a variable
```

Comment:
this comment is at best useless, because that's just how `const` works. it's actually worse than useless to me, because I got tripped up trying to figure out what you meant. please remove all of these types of comments